stable_baselines3/common/base_class.py

```diff
@@ -681,6 +681,12 @@ class OffPolicyRLModel(BaseRLModel):
     :param buffer_size: (int) size of the replay buffer
     :param learning_starts: (int) how many steps of the model to collect transitions for before learning starts
     :param batch_size: (int) Minibatch size for each gradient update
+    :param tau: (float) the soft update coefficient ("Polyak update", between 0 and 1)
```

Comment:
Could clarify here that "tau=1.0" means hard updates (no soft updating)

Reply:
yep, thought the same... but we may change the hyperparameters as a buffer of 1M frames x 4 don't fit on many pc
will see (currently using a buffer size of 5e5 for the tests)